classes/Tag.php

```diff
@@ -345,8 +348,8 @@ private static function deleteProductTags($idProduct, int $langId = null)
             WHERE id_product=' . (int) $idProduct
         ;
         if ($langId) {
-            $removeWhere .= ' AND id_lang =' . (int) $langId;
-            $selectTagsToRemove .= ' AND id_lang =' . (int) $langId;
+            $removeWhere .= ' AND id_lang =' . $langId;
+            $selectTagsToRemove .= ' AND id_lang =' . $langId;
```

Comment:
Why remove the type casting? It is a protection against SQL injection

Reply:
Hi @matks, see the `$langId` declaration type in function signature (`int`|`null`).